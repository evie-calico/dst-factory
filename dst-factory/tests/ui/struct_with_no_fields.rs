use dst_factory::make_dst_factory;

#[make_dst_factory]
struct StructWithNoFields {
    // No fields
}

fn main() {
}
