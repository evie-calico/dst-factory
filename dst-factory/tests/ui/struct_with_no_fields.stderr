error: Struct must have at least one field
 --> tests/ui/struct_with_no_fields.rs:4:27
  |
4 |   struct StructWithNoFields {
  |  ___________________________^
5 | |     // No fields
6 | | }
  | |_^
